src/views/SignUp.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <fwb-alert type="success" v-if="showSuccessAlert">
+    Your form is submitted successfully!
+  </fwb-alert>
+  <fwb-input v-model="name" label="Name" />
```

Comment:
We don't need a name field actually. I'm about to finish the discord integration. We will have a full user object present at all times. We can literally just make it a singular button.